pkg/server/testserver.go

```diff
@@ -115,6 +116,16 @@ func makeTestSQLConfig(st *cluster.Settings, tenID roachpb.TenantID) SQLConfig {
 	return MakeSQLConfig(tenID, base.DefaultTestTempStorageConfig(st))
 }
 
+func initTraceDir(dir string) error {
+	if dir == "" {
+		return nil
+	}
+	if err := os.MkdirAll(dir, 0755); err != nil {
+		return errors.Newf("cannot create trace dir; traces will not be dumped: %+v", err)
```

Comment:
Looking through these errors we're creating, a more idiomatic pattern might be something like:
`errors.Wrap(err, "cannot create trace dir; traces will not be dumped")`

Reply:
done.